.travis.yml

```diff
@@ -28,16 +28,16 @@ install:
       cabal)
         cabal --version
         travis_retry cabal update
-        cabal install --only-dependencies --enable-tests
-        cabal install hlint
+        cabal install -j --only-dependencies --enable-tests
+        cabal install -j hlint
         ;;
     esac
 
 script:
   - |
     case "$BUILD" in
       cabal)
-        cabal configure --enable-tests
+        cabal configure --enable-tests --ghc-options="-j +RTS -A128m -s -RTS"
```

Comment:
Something occurs to me - will the build fail if _any_ of the commands fail?  e.g. like enabling `set -e` in a bash script.